projects/rero/ng-core/src/lib/record/editor/type/remote-typeahead/remote-typeahead.component.ts

```diff
@@ -25,80 +26,83 @@ import { RemoteTypeaheadService } from './remote-typeahead.service';
 
 @Component({
   selector: 'ng-core-remote-typeahead',
+  styles: ['.dropdown-item:hover{ cursor:pointer;}'],
   templateUrl: './remote-typeahead.component.html'
 })
 export class RemoteTypeaheadComponent extends FieldType implements OnInit {
 
+  // COMPONENT ATTRIBUTES =====================================================
   /** Input search string */
   search: string;
-
   /** Loading data */
   typeaheadLoading: boolean;
 
   /** Observable on Suggestions Metadata */
   suggestions$: Observable<Array<SuggestionMetadata | string>>;
-
   /** Template representation of the formControl value. */
   valueAsHTML$: Observable<string>;
+  /** Number of result in suggestions list */
+  private _numberOfSuggestions = 10;
 
+  // GETTER & SETTER ==========================================================
   /** Filters options */
   get filters(): remoteTypeaheadFilters | null {
-    return this.field.templateOptions.remoteTypeahead?.filters;
+    return this._rtOptions?.filters;
   };
 
-  /** Number of result in suggestions list */
-  private _numberOfSuggestions = 10;
-
   /** Remote Typeahead options from the JONSchema */
   private get _rtOptions(): remoteTypeahead {
     return this.field.templateOptions.remoteTypeahead;
   }
 
+  /**
+   * Return the group field when the suggestion as grouped by category.
+   * @returns string - the name of the field in suggestion list containing the group category. null for disabled.
+   */
+  get groupField(): string | null {
+    const enableGF = this._remoteTypeaheadService.enableGroupField(this._rtOptions);
+    return enableGF ? 'group' : null;
```

Comment:
`return this._remoteTypeaheadService.enableGroupField(this._rtOptions) ? 'group' : null;`